packages/material-ui/src/ListItemText/ListItemText.d.ts

```diff
@@ -6,14 +6,42 @@ export interface ListItemTextProps<
   PrimaryTypographyComponent extends React.ElementType = 'span',
   SecondaryTypographyComponent extends React.ElementType = 'p'
 > extends StandardProps<React.HTMLAttributes<HTMLDivElement>, ListItemTextClassKey> {
+  /**
+   * Alias for the `primary` property.
```

Comment:
I have added a new item in the next batch to rename `property` -> `prop`. This is part of https://trello.com/c/AxWGaWEw/1390-replace-property-prop-in-the-react-element-context.